src/components/pages/group/edit/group-rules/group-rules-table/show-less.tsx

```diff
@@ -0,0 +1,32 @@
+import { IconButton } from "@chakra-ui/core";
+import useTranslation from "@configs/i18n/useTranslation";
+import styled from "@emotion/styled";
+import notification, { NotificationType } from "@utils/notification";
+import React from "react";
+
+const Ellipsis = styled.div`
+  width: 30rem;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
```

Comment:
We already have a pre-defined elipsis class that will use webkit line clamping instead of elipsis since it requires fixed width

After our talk I checked you can copy full content by selecting full line so even we can avoid copy button as well

Reply:
MDN example showing you can select full paragraph content
https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp